tictactoe/src/console_io.erl

```diff
@@ -1,11 +1,18 @@
 -module(console_io).
--export([display/1,greet_user/0]).
+-export([	display/1,
+					greet_user/0,
+					display_board/1]).
 -import(instructions, [greet/0]).
 
 
 display(Message) ->
   io:fwrite(Message).
 
+display_board([]) -> board;
+display_board([RowToPrint | RemainingRows]) ->
```

Comment:
Where is the test for this function?

Reply:
This one I wasn't sure how to test, how would you recommend I test something like this?